build.sh

```diff
@@ -69,6 +71,12 @@ for p in ${PACKAGE_FORMATS} ; do
         var
       __ret=$?
      ;;
+     tar)
+      tar \
+        -cvzf cdap-ambari-service-mpack-${PACKAGE_VERSION}-${PACKAGE_ITERATION}.tar.gz \
```

Comment:
i might suggest just naming it `cdap-mpack.tar.gz` ?  Seems more concise to me... maybe we can discuss internally.